### DISCUSSION TITLE
multi-select picker and myql database

### DISCUSSION THREAD

**Discussion by upgreydd** [Budibase/budibase#5436] [Help]
Hi. Which type of data from mysql side represents multi-select picker?

**Comment by upgreydd** [comment]
OK it's `longtext` but when I defined `longtext` on the mysql side and try to use it I'm getting `This Field setting is the wrong data type for this component`. Any idea how to apply `multi-select` type to existing database for specific field?

**Comment by melohagan** [comment]
Hi @Upgreydd 

If you are using a data provider as the options source, then you can use text / longtext fields, but you will still need to give the field a unique name: 

![Screenshot 2022-04-19 at 09 35 26](https://user-images.githubusercontent.com/101575380/163960880-87c127e6-3fc5-4e4c-9fc3-9f247bdde319.png)

![Screenshot 2022-04-19 at 09 36 42](https://user-images.githubusercontent.com/101575380/163961120-2ade8f6d-e613-4985-b58b-d2cf70f5721a.png)

![Screenshot 2022-04-19 at 09 37 15](https://user-images.githubusercontent.com/101575380/163961200-82bafd2b-0f90-497c-9e3b-1cd0dfc57a62.png)
